pymathics/graph/base.py

```diff
@@ -636,12 +638,17 @@ def empty(self):
     def head(self):
         return SymbolGraph
 
+    def is_directed(self):
+        if self.G.is_directed():
+            return not self.mixed
+        return False
+    
     def is_loop_free(self):
         return not any(True for _ in nx.nodes_with_selfloops(self.G))
 
     # networkx graphs can't be for mixed
```

Comment:
can't be for -> cannot be used for

Reply:
I will correct this when I rebase for https://github.com/Mathics3/pymathics-graph/pull/13

Reply:
Excellent. With this baseline, we can go over #13 and see what is failing.